packages/app/src/hooks/useExplainQuery.tsx

```diff
@@ -0,0 +1,36 @@
+import { useQuery } from '@tanstack/react-query';
+
+import { sendQuery } from '@/clickhouse';
+import {
+  ChartConfigWithDateRange,
+  renderChartConfig,
+} from '@/renderChartConfig';
+
+export function useExplainQuery(
+  config: ChartConfigWithDateRange,
+  options?: { enabled?: boolean },
```

Comment:
another style note: I've found that keeping options as something like:
https://github.com/hyperdxio/hyperdx/blob/83f9997c03a31c73fd7dfbf03a83d1a34909a483/packages/app/src/BenchmarkPage.tsx#L87
and then spreading it below

It's helpful as it allows the callers to easily add more react query option overrides in the future. It helps with consistency so all hooks can support the same set of overrides more or less.